pkg/minishift/registration/redhat.go

```diff
@@ -63,8 +64,12 @@ func (registrator *RedHatRegistrator) CompatibleWithDistribution(osReleaseInfo *
 
 // Register attempts to register the system with RHSM
 func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) error {
+	var spinnerView *spinner.Spinner
+
 	if isRegistered, err := registrator.isRegistered(); !isRegistered && err == nil {
-		spinnerView := spinner.New(spinner.CharSets[9], SPINNER_SPEED*time.Millisecond)
+		if !cmdUtil.HasEnabledNoProgressbar {
```

Comment:
I believe we should use `ShowProgressbars` (default `true`) and allow the `ProcessEnvironmentVariables` function to toggle this off. This sentence has a double negation... :-s

Reply:
Ack